.test-infra/jenkins/job_PostCommit_Python_Chicago_Taxi_Example_Flink.groovy

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import CronJobBuilder
+import Docker
+import Flink
+import LoadTestsBuilder
+import PostcommitJobBuilder
+
+def chicagoTaxiJob = { scope ->
+    scope.description('Runs the Chicago Taxi Example on the Flink runner.')
+    commonJobProperties.setTopLevelMainJobProperties(scope, 'master', 120)
+
+    def numberOfWorkers = 5
+
+    Docker publisher = new Docker(scope, LoadTestsBuilder.DOCKER_CONTAINER_REGISTRY)
+    publisher.publish(':sdks:python:container:py2:docker', 'python2.7_sdk')
```

Comment:
I wonder if we should be using a py3 container straight out

Reply:
though maybe then we'd have to run everything in py3 (meaning the target `:portable:py2:chicagoTaxiExample`). LMK what you think.

Reply:
I ran the test in py3 (both SDK harness and the pipeline program) and there are some errors connected with `dill` package. What do you think about leaving it "as is" and creating a jira ticket to move chicago taxi to py3? We will have to do the migration anyway in the nearest future.

Reply:
https://issues.apache.org/jira/browse/BEAM-9154